core/packages/contracts/package.json

```diff
@@ -35,7 +35,10 @@
     "@nomiclabs/hardhat-ethers": "^2.0.2",
     "@nomiclabs/hardhat-etherscan": "^2.1.4",
     "@openzeppelin/contracts": "^4.1.0",
-    "@polkadot/api": "^9.6.2",
+    "@polkadot/api": "9.11.1",
+    "@polkadot/types": "9.11.1",
+    "@polkadot/util": "10.2.3",
+    "@polkadot/util-crypto": "10.2.3",
```

Comment:
Do we need types, util & util-crypto? Not seeing any uses in this PR.

Reply:
yes, I think `types` is required here and I remove the others which is not used https://github.com/Snowfork/snowbridge/blob/b6546280cf2ad67da4cf69702971be50c3013789/core/packages/contracts/scripts/configure-beefy.ts#L5-L9